contrib/upgrade.sh

```diff
@@ -24,7 +27,8 @@
 
 function giteacmd {
   if [[ $sudocmd = "su" ]]; then
-    "$sudocmd" - "$giteauser" -c "$giteabin" --config "$giteaconf" --work-path "$giteahome" "$@"
+    # `-c` accept one string as argument.
+    "$sudocmd" - "$giteauser" -c "$giteabin --config $giteaconf --work-path $giteahome $@"
```

Comment:
Although it was incorrect, the changed line might also have small problems if there are spaces in the arguments.

I do not know how to correctly quote these arguments inside the `"`

Reply:
It's incorrect and throws error in my `OpenWrt`.